opensplice_cmake_module/env_hook/opensplice.bat.in

```diff
@@ -2,6 +2,7 @@ set "OSPL_HOME=@OpenSplice_HOME@"
 
 call "%OSPL_HOME%\release.bat" 1> nul
 
+set "OSPL_URI=file://@CMAKE_INSTALL_PREFIX@/share/opensplice_cmake_module/config/ros_ospl.xml"
```

Comment:
Please don't use `CMAKE_INSTALL_PREFIX` since it break ability to relocate the install space. Use `$AMENT_CURRENT_PREFIX` instead.